src/Header.js

```diff
@@ -12,8 +12,8 @@ class Header extends Component {
     return (
       <header className="header">
         <div className="header-search-cont">
-          <img className="header-svg" src={require('./css/images/ice-cream.svg')}/>
-          <header className="title">iScream</header>
+          <img src={require('./css/images/ice-cream.svg')}/>
+          <h1>iScream</h1>
```

Comment:
I like how you deleted all of the unnecessary classnames and used SASS techniques instead